core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -173,17 +175,20 @@ private void registerContainersForShutdown() {
 
             // Ensure that the default network for this compose environment, if any, is also cleaned up
             ResourceReaper.instance().registerNetworkForCleanup(identifier + "_default");
+            spawnedNetworkIds.add(identifier + "_default");
+
             // Compose can define their own networks as well; ensure these are cleaned up
             dockerClient.listNetworksCmd().exec().forEach(network -> {
                 if (network.getName().contains(identifier)) {
```

Comment:
not related to this PR, but AFAIK we shouldn't use `contains` here, but `startsWith` or whatever the rule for Docker Compose, otherwise if identifier if "a", then it will delete all networks with "a" character in name :D